diff --git a/plugin/SpaceVim.vim b/plugin/SpaceVim.vim
new file mode 100644
index 00000000..015bc231
--- /dev/null
+++ b/plugin/SpaceVim.vim
@@ -0,0 +1,9 @@
+"=============================================================================
+" SpaceVim.vim --- Entry point for SpaceVim installed as a plugin or package
+" Copyright (c) 2016-2022 Wang Shidong & Contributors
+" Author: Wang Shidong < wsdjeg@outlook.com >
+" URL: https://spacevim.org
+" License: GPLv3
+"=============================================================================
+
+execute 'source' fnamemodify(expand('<sfile>'), ':h').'/../main.vim'
